backend/src/main/java/gov/cdc/usds/simplereport/service/PersonService.java

```diff
@@ -176,7 +179,7 @@ protected Specification<Person> buildPersonMatchFilter(
    * @param facilityId If null, then it means across accessible facilities in the whole organization
    * @param pageOffset Pagination offset is zero based
    * @param pageSize How many results to return, zero will result in the default page size (large)
-   * @param isArchived Default is false. true will ONLY show deleted users
+   * @param includeArchived Default is false. true will ONLY show deleted users
```

Comment:
I think this `@param` comment needs to be updated to reflect the new behavior. my understanding is true shows all users and false will continue to only show active users, as before

Reply:
Thanks! Done in [990411b](https://github.com/CDCgov/prime-simplereport/pull/4229/commits/990411be2ee1fd3f44795b114f5b67518b14e7f8)